doc/usecases/elektra_web/UC_duplicating_keys.md

```diff
@@ -0,0 +1,20 @@
+# Use Case: Duplicating keys
```

Comment:
I somehow doubt that it is a common operation outside of the student use case (which is not relevant anymore). I rather think that common use cases are:

- add array member (`#N+1`)
- add a key within a specific namespace for which the specification is given (but no key exists)